__tests__/api.test.js

```diff
@@ -83,6 +83,7 @@ describe(' GET /api/reviews/:review_id', () => {
                     review_id: 1,
                     title: 'Agricola',
                     category: 'euro game',
+                    comment_count: 0,
```

Comment:
Here, we don't want to refactor our original test but instead make a new test.

If it doesn't pass, your original might be too brittle - something we want to avoid. To make sure it's not too brittle, have a look at .toMatchObject or .objectContaining.